### ISSUE TITLE
[bitnami/discourse] Upgrade Discourse to the latest chart version

### ISSUE THREAD

**Issue by agrosu1978** [bitnami/charts#20317]
### Name and Version

bitnami/discourse 12.0.3

### What architecture are you using?

amd64

### What steps will reproduce the bug?

I run the following command to upgrade the discourse deployment with the lastest chart version:
`helm upgrade -n rws <RELEASE_NAME> --set global.postgresql.auth.password=$PASSWORD --set auth.password=$DISCOURSE_PASSWORD  bitnami/discourse`

The problem is that this chart is using the packaged postgresql chart version 13.0.0 which installs postgres 16.0 . Thus, the postgres doesn't start anymore, stating that the database is created by postgres 14.4 (previous version). 

The guys from discourse told me to try to get help from here, because discourse does not support postgres 16.0 yet. Is there a way to force the discourse chart to use the postgresql-11.6.16 chart (which installs  postgres 14.4)

Thank you for all your help !

### What is the expected behavior?

Postgresql pod should start with the existent database

### What do you see instead?

Postgresql version 16 complains that it can't use a database created with version 14.4

**Comment by javsalgar** [comment]
Hi,

One option that comes to mind is that you replace the `postgresql.image.tag` value with one of the 14 branch. Could you try that?

**Comment by agrosu1978** [comment]
> Hi,
> 
> One option that comes to mind is that you replace the `postgresql.image.tag` value with one of the 14 branch. Could you try that?

Yes, it worked, thank you ! I tried this before yesterday, but some strange things were happening, like two pods of discourse in parallel, one running and one in ContainerCreating state. The one running was not able to connect to Postgres 14.4. But it seems I had to delete them, and wait for a bit longer for things to come in place.
So yes, I confirm that ```--set postgresql.image.tag=14.4.0``` is working
